### PR TITLE
Update title.json

### PR DESCRIPTION
**Pull request by J1mBob**
Added granted slots and stats to huge ships, added images.

### CODE REVIEW

File: data/upgrades/title.json [hunk 1]
```diff
@@ -546,7 +546,13 @@
         "title": "Assailer",
         "type": "Title",
         "slots": ["Title"],
+        "grants": [
+          { "type": "slot", "value": "gunner", "amount": 1 },
+          { "type": "stat", "value": "hull", "amount": 2 },
+          { "type": "stat", "value": "shields", "amount": -2 }
+        ],
         "ffg": 769,
+        "image": "https://infinitearenas.com/xw2browse/epic/assailer.PNG",
```
Comment (guidokessels) [inline data/upgrades/title.json:555]: Hmm these are very bad quality pictures :( I wonder if there are any better ones available. I can't seem to find any

Comment (J1mBob) [reply data/upgrades/title.json:555]: I agree and hope someone has better.

Comment (danrs) [reply data/upgrades/title.json:555]: This image now 404s anyway

**Review by danrs — CHANGES_REQUESTED** [review]
Image URLs should use https://github.com/CrazyVulcan/x-wing2.0-project-goldenrod instead of the FFG CDN

https://github.com/CrazyVulcan/x-wing2.0-project-goldenrod src/image/En for cards and src/images/Art for art

**Comment by danrs** [comment]
Closing due to inactivity

**Pull request closed without merging**